docker/mesos-cloud.conf

```diff
@@ -0,0 +1,4 @@
+[mesos-cloud]
+  http-client-timeout = 5s
+  state-cache-ttl     = 20s
+    
```

Comment:
Without having the complete overview, can't you put the mesos:5050 line here as well? It's constant after all.